openmeter/customer/adapter/customer.go

```diff
@@ -294,6 +293,16 @@ func (a *adapter) GetCustomer(ctx context.Context, input customerentity.GetCusto
 				WithSubjects(func(query *entdb.CustomerSubjectsQuery) {
 					query.Where(customersubjectsdb.IsDeletedEQ(false))
 				}).
+				WithApps(func(acq *entdb.AppCustomerQuery) {
```

Comment:
The alternative would be to call app service, which would lead to waterfall DB queries and circular dependencies.